src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected LocalDateTime date;
+
+    public Deadline(String description, String date) {
```

Comment:
Perhaps a method could have a header comment to inform users about what the method does.

Same suggestion for the other methods both in this file and other .java files